demos/CompositeUI-MVC/Divergent.ITOps.ViewModelComposition/DynamicViewModel.cs

```diff
@@ -77,9 +84,28 @@ public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, o
             return false;
         }
 
+        public override IEnumerable<string> GetDynamicMemberNames()
+        {
+            return base.GetDynamicMemberNames()
+                .Union(properties.Keys)
+                .Union(new[] { "RaiseEventAsync", "RaiseEvent" });
```

Comment:
agree. In this specific sample that method in never called. Anyway do you mean something like this:

```
foreach (var item in base.GetDynamicMemberNames())
{
      yield return item;
}

foreach (var item in properties.Keys)
{
      yield return item;
}

yield return "RaiseEventAsync";
```